sys/include/net/sock/udp.h

```diff
@@ -367,6 +367,11 @@ typedef struct {
  * @pre `(sock != NULL)`
  * @pre `(remote == NULL) || (remote->port != 0)`
  *
+ * @warning If you create a socket you are responsible for receiving messages
+ *          sent to it by calling @ref sock_udp_recv.
```

Comment:
Maybe also add the reason?
```suggestion
 *          sent to it by calling @ref sock_udp_recv. Otherwise, the packet queue of the @p sock may congest.
```

Reply:
(of course also fix the line-length afterwards ;-))